cli/dcoscli/package/main.py

```diff
@@ -450,10 +450,14 @@ def _search(json_, query):
     results = [index_entry.as_dict()
                for index_entry in package.search(query, config)]
 
-    emitting.publish_table(emitter,
-                           results,
-                           tables.package_search_table,
-                           json_)
+    if any([len(element.get('packages', []))
```

Comment:
change this to `any(result['packages'] for result in results)`

The pythonic way to check for a nonempty list is to use the list itself as the predicate.